gef-ui/server/controllers/application-submission/index.js

```diff
@@ -64,7 +69,7 @@ const postApplicationSubmission = async (req, res, next) => {
     dealId,
     submissionType,
     status: currentStatus,
-    unissuedToIssued: hasChangedToIssued(application),
+    issuedFacility,
```

Comment:
is this a flag? If so could we call it `hasIssuedFacility`? As it stands,it looks like it will be a facility object - which is great if that's what it is?

Reply:
Have updated the to `hasIssuedFacility` as it is flag.